src/BlockEntities/BrewingstandEntity.h

```diff
@@ -119,6 +126,9 @@ class cBrewingstandEntity :
 	/** Amount of ticks that the current item has been brewed */
 	short m_TimeBrewed;
 
+	/** The fuel for the brewing stand */
+	short m_Fuel;
```

Comment:
(and with @Pokechu22 's description in the doxy-comment and somewhere in the APIDoc)